ci/cloudbuild/run_single_test.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+#
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+export REGION_ID='uc'
+
+d=$(pwd)
+PROJECT=$(basename ${d})
+
+# Activate mocha config
+export MOCHA_REPORTER_OUTPUT=${PROJECT}_sponge_log.xml
+export MOCHA_REPORTER=xunit
+
+# Install dependencies
+npm install
```

Comment:
Is this where we would inject any service account or any other variables to run the actual tests? Or, where would it live?

Reply:
The service account is attached to the build. See the trigger creation script.